test/com/amazon/ionschema/IonSchemaTestRunner.kt

```diff
@@ -60,7 +60,7 @@ class IonSchemaTestRunner(
             .filter { it.path.endsWith(".isl") }
             .filter { !blacklist.contains(it.path) }
             .forEach { file ->
-                val testName = file.path.substring(base.length + 1, file.path.length - ".isl".length)
+                val testName = file.path.substring(base.length + 1, file.path.length)
```

Comment:
I think this will cause all of the test names in the output to include the `.isl` suffix as well. If you make a separate value (`testFile`?) and pass that to the `SchemaImpl` constructors downstream, we can update the tests' behavior without changing their names.

```suggestion
                val testFile = file.path.substring(base.length + 1, file.path.length)
                val testName = file.path.substring(base.length + 1, file.path.length - ".isl".length)
```